eras/allegra/impl/cddl-files/allegra.cddl

```diff
@@ -1,346 +1,339 @@
-
-auxiliary_data = metadata
-                  / [transaction_metadata : metadata
-                    , auxiliary_scripts : auxiliary_scripts]
 
-auxiliary_scripts = [* native_script]
-
-block = [header
-        , transaction_bodies : [* transaction_body]
-        , transaction_witness_sets : [* transaction_witness_set]
-        , auxiliary_data_set : {* transaction_index => auxiliary_data}]
+block = 
+  [ header                 
+  , transaction_bodies       : [* transaction_body]                   
+  , transaction_witness_sets : [* transaction_witness_set]            
+  , auxiliary_data_set       : {* transaction_index => auxiliary_data}
+  ]
 
-bootstrap_witness = [public_key : $vkey
-                    , signature : $signature
-                    , chain_code : bytes .size 32
-                    , attributes : bytes]
 
-certificate = [stake_registration
-               // stake_deregistration
-               // stake_delegation
-               // pool_registration
-               // pool_retirement
-               // genesis_key_delegation
-               // move_instantaneous_rewards_cert]
+header = [header_body, body_signature : kes_signature]
 
-coin = uint
+header_body = 
+  [ block_number      : uint        
+  , slot              : uint        
+  , prev_hash         : hash32/ nil 
+  , issuer_vkey       : vkey        
+  , vrf_vkey          : vrf_vkey    
+  , nonce_vrf         : vrf_cert    
+  , leader_vrf        : vrf_cert    
+  , block_body_size   : uint .size 4
+  , block_body_hash   : hash32      
+  , operational_cert
+  , protocol_version
+  ]
 
-credential = [0, addr_keyhash // 1, script_hash]
 
-; This too has been introduced in Shelley as a backport from Alonzo.
-; 
-delta_coin = int
+hash32 = bytes .size 32
 
-dns_name = text .size (0 .. 64)
+vkey = bytes .size 32
 
-epoch = uint
+vrf_vkey = bytes .size 32
 
-genesis_delegate_hash = $hash28
+vrf_cert = [bytes, bytes .size 80]
 
-genesis_hash = $hash28
+operational_cert = 
+  ( hot_vkey        : kes_vkey 
+  , sequence_number : uint     
+  , kes_period      : uint     
+  , sigma           : signature
+  )
 
-header = [header_body, body_signature : $kes_signature]
+kes_vkey = bytes .size 32
 
-header_body = [block_number : uint
-              , slot : uint
-              , prev_hash : $hash32 / nil
-              , issuer_vkey : $vkey
-              , vrf_vkey : $vrf_vkey
-              , nonce_vrf : $vrf_cert
-              , leader_vrf : $vrf_cert
-              , block_body_size : uint .size 4
-              , block_body_hash : $hash32
-              , operational_cert
-              , protocol_version]
+signature = bytes .size 64
 
-int64 = -9223372036854775808 .. 9223372036854775807
+protocol_version = (major_protocol_version, uint)
 
-ipv4 = bytes .size 4
+major_protocol_version = 1 .. 3
 
-ipv6 = bytes .size 16
+kes_signature = bytes .size 448
 
-major_protocol_version = 1 .. 3
+;Allegra transaction body adds the validity interval start at index 8
```

Comment:
Comments could use a space. Any chance cuddle could be adjusted to accommodate that?
```suggestion
; Allegra transaction body adds the validity interval start at index 8
```